Sources/AppStoreConnectCLI/Model/Profile.swift

```diff
@@ -7,15 +7,12 @@ import SwiftyTextTable
 
 extension Model.Profile {
     init(_ apiProfile: AppStoreConnect_Swift_SDK.Profile) {
-        self.init(apiProfile.attributes!)
-    }
-
-    init(_ attributes: AppStoreConnect_Swift_SDK.Profile.Attributes) {
+        let attributes = apiProfile.attributes!
         self.init(
             name: attributes.name,
             platform: attributes.platform?.rawValue,
             profileContent: attributes.profileContent,
-            uuid: attributes.uuid,
```

Comment:
The UUID is important too in profiles.  As it is usually the filename used to save the profile.

Reply:
As UUID can't be used to get profile, how about showing ID and UUID together?
```
asc profiles list
+------------+------------------------------+----------+--------+---------------+----------------------+----------------------+
| ID         |  UUID         | Name                         | Platform | State  | Type          | Created Date         | Expiration Date      |
+------------+------------------------------+----------+--------+---------------+----------------------+----------------------+
```

Reply:
Showing them together is fine.

Reply:
✅